controllers/replicationsource.go

```diff
@@ -131,18 +131,20 @@ func (r *VolumeSnapshotBackupReconciler) setVSBRepSourceStatus(log logr.Logger)
 		if err != nil {
 			return false, err
 		}
-		reconCondition := metav1.Condition{}
-		reconConditionProgress := metav1.Condition{}
+
+		reconConditionStatus := metav1.Condition{}
+		reconConditionReason := metav1.Condition{}
+
 		for i := range repSource.Status.Conditions {
-			if repSource.Status.Conditions[i].Type == "Reconciled" {
-				reconCondition = repSource.Status.Conditions[i]
+			if repSource.Status.Conditions[i].Status == metav1.ConditionFalse {
+				reconConditionStatus = repSource.Status.Conditions[i]
 			}
 			if repSource.Status.Conditions[i].Reason == volsyncv1alpha1.SynchronizingReasonSync {
```

Comment:
IS the check for SynchronizingReasonSync Correct ? Should it be `WaitingForManual` I am unsure.

Reply:
`waitingForManual` only appears when one completion of replicationSource has finished.